pkg/monitoring/metrics.go

```diff
@@ -0,0 +1,87 @@
+package monitoring
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	v1 "github.com/rancher/backup-restore-operator/pkg/apis/resources.cattle.io/v1"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	backupCount = promauto.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "bro_backup_count",
+			Help: "Number of existing Rancher Backup CRs",
+		})
+
+	backup = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "bro_backup",
+			Help: "Details on a specific Rancher Backup CR",
+			// add labels showing encryption type and current status
+		}, []string{"name", "resourceSetName", "retentionCount", "backupType", "filename", "storageLocation", "nextSnapshot", "lastSnapshot"})
+
+	backupLastProcessed = promauto.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "bro_backup_last_processed",
+			Help: "Unix time of when the last Backup was processed (in seconds)",
```

Comment:
Metric names should encode the units per naming conventions https://prometheus.io/docs/practices/naming/